code-submissions-clustering-core/src/main/kotlin/org/jetbrains/research/code/submissions/clustering/util/LoadUtil.kt

```diff
@@ -26,14 +26,15 @@ fun <T> DataFrame<*>.loadGraph(context: SubmissionsGraphContext<T>): Submissions
                     Submission(
                         info = SubmissionInfo(getValue(id), getValue(quality)),
                         stepId = getValue(step_id),
-                        code = getValue(code)
+                        code = getValue(code).replace("\r\n", "\n")
```

Comment:
Should we also replace `\r`? 
[see](https://stackoverflow.com/questions/3776923/how-can-i-normalize-the-eol-character-in-java)

Also I'd suggest moving this code into a separated function